tests/test_bootstrap_field_input_text.py

```diff
@@ -8,6 +8,10 @@ class CharFieldTestForm(forms.Form):
     test = forms.CharField()
 
 
+class CharFieldRequiredTestForm(forms.Form):
+    test = forms.CharField(min_length=1)
```

Comment:
This does not perform an assert. Is that intentional?

Reply:
This test form was created to produce a form that will fail a `form.is_valid()` check. The response can then be rendered to check the correct HTML is returned for the failing form field. It's used [here](https://github.com/tom-price/django-bootstrap5/blob/9c1ee5b59dd6b7f42eceaa941392b324ed1f9329/tests/test_bootstrap_field_input_text.py#L155-L173) to do so.